definitions/procedures/pulp/remove.rb

```diff
@@ -0,0 +1,90 @@
+module Procedures::Pulp
+  class Remove < ForemanMaintain::Procedure
+    metadata do
+      description 'Remove pulp2'
+      for_feature :pulp2
+    end
+
+    def sys
+      ForemanMaintain::Utils::SystemHelpers.new
+    end
+
+    def pulp_data_dirs
+      [
+        feature(:pulp2).data_dir + '/published',
+        feature(:pulp2).data_dir + '/content',
+        feature(:pulp2).data_dir + '/importers',
+        feature(:pulp2).data_dir + '/uploads'
+      ]
+    end
+
+    def pulp_packages
+      [
+        'pulp-server', 'python-pulp-streamer', 'pulp-puppet-plugins',
+        'python-pulp-rpm-common', 'python-pulp-common',
+        'pulp-selinux', 'python-pulp-oid_validation',
+        'python-pulp-puppet-common', 'python-pulp-repoauth',
+        'pulp-rpm-plugins'
+      ]
```

Comment:
@ehelms @ekohl curious if ya'll have any thoughts around other things we should clean up?

Reply:
we potentially could also remove these dependencies: 

python-blinker python-celery python-django   python-isodate python-ldap python-mongoengine python-nectar python-oauth2 python-pymongo

There are a couple more deps that also cause other things to be removed, such as  'python-httplib2'  causes ansible to be removed, and python-gofer/python-qpid causes katello-agent to be removed.    I kinda lean towards not removing any of these, but curious what others think

Reply:
If we are cleaning up mongo data, we should clean up rh-mongo34 packages as well

Reply:
If its something we distribute, via a Pulp repository or Katello repository, thats not longer needed I say we remove it.

Reply:
@jjeffers want to add those to the list?